scripts/processAndDownloadContracts.ts

```diff
@@ -17,21 +17,25 @@ async function processContracts() {
 			...await getAaveV2Misc()
 		]
 	const output = `
-
-const contractData: [string, string, string, string | null][] = ${JSON.stringify(contractData.map((x) => [addressString(x.address), x.data.name, x.data.protocol, x.data.logoURI]), null, 1)}
+import * as contractData from './contractMetadata.json';
 export type ContractDefinition = {
-    name: string,
-    logoURI: string | undefined,
-    protocol: string | undefined
+	name: string,
+	logoURI: string | undefined,
+	protocol: string | undefined
```

Comment:
Consider using the TS "optional" syntax rather than a union of undefined.  This allows the item to be missing, rather than being present but set to `undefined`.
```suggestion
	logoURI?: string,
	protocol?: string,
```